tests/v2/validation/rbac/rbac_test.go

```diff
@@ -335,6 +335,72 @@ func (rb *RBTestSuite) ValidateAddCMAsProjectOwner() {
 
 }
 
+func (rb *RBTestSuite) ValidateAddPOsAsProjectOwner() {
+	createProjectAsCO, err := createProject(rb.client, rb.cluster.ID)
+	require.NoError(rb.T(), err)
+	rb.standardUserCOProject = createProjectAsCO
+
+	log.Info("Additional Testcase4 - Validating if Project Owner can add another Project Owner ")
+	errUserRole := users.AddProjectMember(rb.standardUserClient, rb.standardUserCOProject, rb.additionalUser, roleProjectOwner)
+	require.NoError(rb.T(), errUserRole)
+	rb.additionalUserClient, err = rb.additionalUserClient.ReLogin()
+
+	addNewUserAsPO, err := createUser(rb.client, standardUser)
+	addNewUserAsPOClient, err := rb.client.AsUser(addNewUserAsPO)
+	require.NoError(rb.T(), err)
+
+	errUserRole2 := users.AddProjectMember(rb.additionalUserClient, rb.standardUserCOProject, addNewUserAsPO, roleProjectOwner)
+	require.NoError(rb.T(), errUserRole2)
+	addNewUserAsPOClient, err = addNewUserAsPOClient.ReLogin()
+
+	//Check user is associated with the project
```

Comment:
Can this comment be removed? Apologies, I meant all comments in my earlier review.